### PR TITLE
Zipfldr drag-drop test

### PR DESCRIPTION
**Pull request by learn-more**
Tests showing some properties of zipfldr IDataObject

JIRA issue: [CORE-16241](https://jira.reactos.org/browse/CORE-16241)

### CODE REVIEW

File: modules/rostests/apitests/browseui/IACLCustomMRU.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  * PROJECT:     ReactOS api tests
- * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
  * PURPOSE:     Test for IACLCustomMRU objects
  * COPYRIGHT:   Copyright 2017 Mark Jansen (mark.jansen@reactos.org)
```
Comment (katahiromz) [inline modules/rostests/apitests/browseui/IACLCustomMRU.cpp]: update year

File: modules/rostests/apitests/zipfldr/zipfldr.cpp [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+        return FALSE;
+    }
+    Success = WriteFile(Handle, Data, Size, &dwWritten, NULL);
+    ok(Success == TRUE, "WriteFile failed with %lu\n", GetLastError());
+    ok(dwWritten == Size, "WriteFile wrote %lu bytes instead of %lu\n", dwWritten, Size);
+    CloseHandle(Handle);
+    return Success && (dwWritten == Size);
+}
+
+BOOL extract_resource(WCHAR* Filename, LPCWSTR ResourceName)
```
Comment (katahiromz) [inline modules/rostests/apitests/zipfldr/zipfldr.cpp]: add static

File: modules/rostests/apitests/zipfldr/zipfldr.cpp [hunk 3]
```diff
@@ -0,0 +1,536 @@
...
+{
+    WCHAR workdir[MAX_PATH];
+    GetTempPathW(_countof(workdir), workdir);
+    StringCchPrintfW(Filename, MAX_PATH, L"%sTMP%u.zip", workdir, GetTickCount() & 0xffff);
+
+    HMODULE hMod = GetModuleHandleW(NULL);
+    HRSRC hRsrc = FindResourceW(hMod, ResourceName, MAKEINTRESOURCEW(RT_RCDATA));
+    ok(!!hRsrc, "Unable to find %s\n", wine_dbgstr_w(ResourceName));
+    if (!hRsrc)
```
Comment (katahiromz) [inline modules/rostests/apitests/zipfldr/zipfldr.cpp]: add skip

File: modules/rostests/apitests/zipfldr/zipfldr.cpp [hunk 4]
```diff
@@ -0,0 +1,536 @@
...
+        return FALSE;
+
+    HGLOBAL hGlobal = LoadResource(hMod, hRsrc);
+    DWORD Size = SizeofResource(hMod, hRsrc);
+    LPVOID pData = LockResource(hGlobal);
+
+    ok(Size && !!pData, "Unable to load %s\n", wine_dbgstr_w(ResourceName));
+    if (!Size || !pData)
+        return FALSE;
```
Comment (katahiromz) [inline modules/rostests/apitests/zipfldr/zipfldr.cpp]: add skip

File: modules/rostests/apitests/zipfldr/zipfldr.cpp (hunk 3 above)
Comment (learn-more) [reply modules/rostests/apitests/zipfldr/zipfldr.cpp]: There is already an error indicating why the test aborted.

**Comment by learn-more** [comment]
Will revisit with an updated test later

**Pull request closed without merging**